dicoogle/src/main/resources/webapp/js/actions/pluginActions.js

```diff
@@ -0,0 +1,4 @@
+import Reflux from 'reflux';
+export const PluginActions = {
+  get: Reflux.createAction()
```

Comment:
We can export actions individually instead: `export const get = Reflux.createAction();`